src/components/VideoCardHome/VideoCardHome.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useContext } from 'react';
+// import { Image } from 'semantic-ui-react';
+import { Card, CardContent, CardMedia, Typography } from '@material-ui/core';
+import { ThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+import { Link } from 'react-router-dom';
+
+import moment from 'moment';
+
+import VideosContext from '../../context/VideosContext';
+import ThemeContext from '../../context/ThemeContext';
+
+const darkTheme = createMuiTheme({
```

Comment:
Please separate the logic of styles in a different file for example: VideoCardGome.styles.js